src/oscar/apps/dashboard/vouchers/views.py

```diff
@@ -4,13 +4,12 @@
 from django.contrib import messages
 from django.db import transaction
 from django.http import HttpResponse, HttpResponseRedirect
-from django.shortcuts import get_object_or_404
+from django.shortcuts import get_object_or_404, redirect
 from django.urls import reverse
 from django.utils import timezone
 from django.utils.translation import gettext_lazy as _
 from django.views import generic
 
-from oscar.apps.voucher.utils import get_offer_name
```

Comment:
@jwayodi `get_offer_name` is no longer used - I think we can remove the method entirely?